server/dbutil/dbutil_test.go

```diff
@@ -0,0 +1,57 @@
+package dbutil
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type DBUtilSuite struct {
+	suite.Suite
+}
+
+func (suite *DBUtilSuite) TestMD5() {
+	hashes := [][]string{
+		{"test string", "6f8db599de986fab7a21625b7916589c"},
+		{"Здравствуйте", "66a2e20820c3e976765ccb17b1b7adca"},
+		{"a multiline\nstring!", "9beef1614897510967755a19341e730d"}}
+
+	for _, hash := range hashes {
+		assert.Equal(suite.T(), hash[1], md5hash(hash[0]))
+	}
+}
+
+func readFile(filename string) string {
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		panic(err)
```

Comment:
`panic` is not needed here. The `readFile` func could return `(string, error)`, and the error would be validated with `assert.NoError(err)`. That way the tests will break gracefully.

Reply:
I'm going to make the change you propose, because I see your point, but at the same time those files are part of the repo and the test.
If the file is not there because I deleted it accidentally, doesn't it make sense to return a severe failure (`panic`) instead of a failed test?

Reply:
Panics are always the last last last option.
In my opinion, for the purpose of a test, it's enough with failing and keep running the other tests.